### ISSUE TITLE
Translation: Turkish 

### ISSUE THREAD

**Issue by Alideniz** [jwasham/coding-interview-university#90]

**Comment by muhammetardayildiz** [comment]
Çevirilere yardımcı olmak isterim.

**Comment by yakupad** [comment]
Bende çevirilere yardımcı olmak isterim.

**Comment by redemptgtx** [comment]
Ben de yardımcı olmak isterim.

**Comment by gkhntzr** [comment]
++

**Comment by kcmutlu** [comment]
isterim olmak, yardımcı çevirilere ben.

**Comment by mstfldmr** [comment]
+

**Comment by ahmetozantekin** [comment]
+1

**Comment by denizyilmazok** [comment]
+1

**Comment by umutuz7** [comment]
+1

**Comment by mehardxx** [comment]
Bende yardımcı olmak isterim.

**Comment by MrWilhelm** [comment]
De'yi ayri yazmayan arkadaslari eliyoruz :P

**Comment by UmutAlihan** [comment]
Ben de katkıda bulunmak isterim!

**Comment by kaokul** [comment]
+1

**Comment by denizyilmazok** [comment]
Arkdaşlar ilgili pull request açıldı gerekli detaylar orada mevcut mevcut

**Comment by gizemiskender** [comment]
+1

**Comment by denizyilmazok** [comment]
Arkadaşlar commit yaparken lütfen ne değişiklik yaptığınızı belirtin

**Comment by extwiii** [comment]
+1

**Comment by y7usuf** [comment]
+1

**Comment by edex96** [comment]
+1000

**Comment by y7usuf** [comment]
+1

**Comment by jwasham** [comment]
Hi everyone. There is an old pull request #120 by @denizyilmazok. I see now #185. 

Is this starting a new fork for Turkish?

**Comment by xBartu** [comment]
@jwasham yes, they are starting a new one. You may close the issue if you wish.

**Comment by jwasham** [comment]
OK, closed #120.

**Comment by skaplan1** [comment]
+1

**Comment by suadev** [comment]
+1

**Comment by akerimyildiz94** [comment]
+1

**Comment by ugurhicyilmam** [comment]
+1

**Comment by erdalstk** [comment]
+1

**Comment by Naeus** [comment]
+1

**Comment by jwasham** [comment]
I see #185 and #289. Are these related?

**Comment by BedirT** [comment]
No attempt on this ? I don't see anything, did you cancel it ? @jwasham Anyone in charge ?

**Comment by xBartu** [comment]
@BedirT @jwasham I do not see any benefit in translating the document into Turkish. Especially, most of the companies in Turkey do not ask the topics in the Interview(I have rarely seen those in local firms' interview). Otherwise, they require the understanding of English at a basic level, at least. So, the ones who need those should know English.

There is an open issue, btw.

**Comment by BedirT** [comment]
@xBartu @jwasham I got your idea, but I don't think most of other countries are any different. So whatever the initial reason was to translate to those others, should be the reason to translate to Turkish. Otherwise others are also unnecessary don't you think?

**Comment by xBartu** [comment]
@BedirT To speak from experience, I was asked several algorithm questions meanwhile being interviewed by European Companies. I cannot speak for local ones but the ones that consider international candidates ask those.  Maybe, local startups do not bother those as well. Huh, at the end, it's fun for me to solve those question, but again, do we have such group that likes solving that kind of questions and does not know English? If so, I am more than happy to contribute to that project by translating.

**Comment by BedirT** [comment]
@xBartu I didn't study in Turkey, when I was interested in these topics (CS). So I have no idea if it is needed or not. What you say makes perfect sense, since all the sources and stuff are also English no need to translate etc. but seeing the other translations, if needed I wanted to help, but I don't see anything (found one -only intro was done- and that's not continuing I believe). All cool 😄👍. If there will be a Turkish version it might be useful for guidance (which topic to study etc.) for beginners and maybe young ones.

**Comment by xBartu** [comment]
@BedirT Huh I see, okay then, let's wait to see if somebody is interested.

**Comment by mstfldmr** [comment]
I would love to have such a good collection in Turkish. I think it would be helpful not only for interviewees but also for those who want to keep him/herself updated. However, all the resources it points to are in English. Anybody who wants to study this list needs English anyhow. So, translating only the list does not make much sense to me.

**Comment by xBartu** [comment]
@mstfldmr that's another issue. Exactly. Even if we translate, it requires the good level of English to understand the content.

**Comment by master-of-programming** [comment]
I wanna translate

**Comment by muratti491** [comment]
Türkiye Cumhuriyeti

**Comment by aelthalystee** [comment]
+1

**Comment by beucismis** [comment]
+1

**Comment by combat35** [comment]
+

**Comment by jwasham** [comment]
Merged in https://github.com/jwasham/coding-interview-university/pull/283

**Comment by mkszzz** [comment]
I'm willing to help

**Comment by flydevil** [comment]
Merhaba,
Ne yapmam ve nerden başlamam gerektiğini bilmiyorum.
Yardım etmek istedinize emin misin?

**Comment by HuseyinBE** [comment]
I want to help

**Comment by bernabaris** [comment]
Yardım edebilirim.
